CoreDataAccessView/test/unit/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/clean/RemoveNodesPluginNGTest.java

```diff
@@ -87,33 +88,32 @@ public void testCreateParameters() {
      * @throws InterruptedException
      * @throws PluginException
      */
-    @Test(expectedExceptions=PluginException.class)
+    @Test(expectedExceptions = PluginException.class)
     public void testIdentifierAttributesNotFound() throws InterruptedException, PluginException {
         System.out.println("identifierAttributesNotFound");
         final RemoveNodesPlugin instance = new RemoveNodesPlugin();
-        final PluginParameters parameters = instance.createParameters(); 
-                
-        VisualConcept.VertexAttribute.SELECTED.ensure(graph);
+        final PluginParameters parameters = instance.createParameters();
+        final DefaultPluginInteraction interaction = mock(DefaultPluginInteraction.class);
         
-        //Method should throw exception as Identifier attribute is not enabled on vertex
-        PluginExecution.withPlugin(instance).withParameters(parameters).executeNow(graph);
+        vertexIdentifierAttribute = VisualConcept.VertexAttribute.IDENTIFIER.ensure(graph);
```

Comment:
is this one meant to be `SELECTED` given its meant to be testing the absence of the `IDENTIFIER` attribute?